drltrace/drltrace_frontend.cpp

```diff
@@ -54,28 +54,41 @@
     exit(1); \
 } while (0)
 
+#define DRLTRACE_INFO(level, msg, ...) do { \
+    if (op_verbose.get_value() >= level) {\
+        fprintf(stderr, "INFO: " msg "\n", ##__VA_ARGS__);    \
+        fflush(stderr); \
+    }\
+} while (0)
+
 static droption_t<std::string> op_logdir
 (DROPTION_SCOPE_ALL, "logdir", ".", "Log directory to print library call data",
```

Comment:
I don't see an XXX comment to use drfront_appdata_logdir() as "." will fail in several situations (Android root, Windows install into ProgFiles, etc., as already mentioned in earlier comments)?